tests/integ/scala/test_dataframe_copy_into.py

```diff
@@ -309,11 +309,11 @@ def test_copy_csv_transformation(session, tmp_stage_name1, trans_columns):
             session.table(table_name),
             [
                 Row("1", "one", "1.2"),
-                Row("2", "two", "2.2"),
                 Row("1.2", "one", "1"),
+                Row("2", "two", "2.2"),
                 Row("2.2", "two", "2"),
             ],
-            sort=False,
+            sort=True,
```

Comment:
This should be the reason for the test to be flaky.

Reply:
Would you like to remove @pytest.mark.xfail on this test to see whether it fixes the flaky tests?

Reply:
Removed